source/lex.tex

```diff
@@ -1893,7 +1893,7 @@
 \end{example}
 The syntactic non-terminal preceding the \grammarterm{ud-suffix} in a
 \grammarterm{user-defined-literal} is taken to be the longest sequence of
-characters that could match that non-terminal.
+characters that would match that non-terminal.
```

Comment:
(Same here as above. If the above is OK, so is this.)

Reply:
```suggestion
characters that match the syntax of that non-terminal.
```